pmm-app/src/pmm-dbaas/components/DBCluster/EditDBClusterModal/DBClusterAdvancedOptions/DBClusterAdvancedOptions.tsx

```diff
@@ -0,0 +1,137 @@
+import React, {
```

Comment:
If there are no changes from the other DBClusterAdvancedOptions component there's no need to duplicate. Just remove this and the tests too will simplify this a lot.

Reply:
Decided to discuss it on a next sync call